src/dso_api/settings.py

```diff
@@ -142,20 +142,17 @@
 }
 
 if CLOUD_ENV.lower().startswith("azure"):
-    with open("/mnt/secrets-store/mdbdataservices-read") as secrets_file:
```

Comment:
Is it possible to trace back from the commit messages why this was done this way?

Reply:
I've updated the commit message to be more verbose on the why.

Reply:
The current implementation comes from https://github.com/Amsterdam/dso-api/pull/355, which probably overlooked that the env var `PGPASSWORD` is exported in the k8s manifest in addition to volume mounting it.